backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/v2ImageUploadController/V2ImageUploadControllerTest.kt

```diff
@@ -106,7 +106,7 @@ class V2ImageUploadControllerTest : AbstractV2ImageUploadControllerTest() {
 
     performAuthDelete("/v2/image-upload/$idsToDelete", null).andIsOk
     val rest = imageUploadService.find(list.map { it.id }.toSet())
-    assertThat(rest).isEqualTo(list.stream().skip(10).collect(Collectors.toList()))
+    assertThat(rest.sortedBy { it.id }).isEqualTo(list.stream().skip(10).collect(Collectors.toList()))
```

Comment:
Out of scope formatting change.

Reply:
Not a format change, `sortBy` added to keep the list sorted. Otherwise, the test failed for me.

Reply:
OK, sorry.